### PR TITLE
existing qa bug fix

### PR DESCRIPTION
**Pull request by ruchitamalviya**

### CODE REVIEW

File: includes/ets_admin_qa_function.php [hunk 1]
```diff
@@ -187,7 +187,7 @@ public function product_tab_admin_qa( $tabs ){
   	public function product_panels_scripts_ui(){    
 		wp_register_script(
 			'jquery-ui-sortable', 
-			array( 'jquery' )
+			array( 'jquery' ),'1.0'
```
Comment (geek92sunny) [inline includes/ets_admin_qa_function.php]: Move to next line, like other parameters.

File: includes/ets_admin_qa_function.php [hunk 2]
```diff
@@ -557,7 +557,7 @@ public function admin_woo_qa_script() {
 		    wp_register_script(
 				'ets_woo_qa_admin_script',
 				ETS_WOO_QA_PATH . 'asset/js/ets_woo_qa_admin_script.js',
-				array('jquery')
+				array('jquery'), '1.0'
```
Comment (geek92sunny) [inline includes/ets_admin_qa_function.php]: Move to next line.

File: includes/ets_user_qa_function.php [hunk 3]
```diff
@@ -516,7 +516,7 @@ public function qa_plugin_script() {
 	public function qa_plugin_style() {
 		wp_register_style(
 		    'ets_woo_qa_style_css',
-		    ETS_WOO_QA_PATH. 'asset/css/ets_woo_qa_style.css'
+		    ETS_WOO_QA_PATH. 'asset/css/ets_woo_qa_style.css', [] , '1.0'
```
Comment (geek92sunny) [inline includes/ets_user_qa_function.php:519]: Next line.

**Comment by ruchitamalviya** [comment]
done

**Review by geek92sunny — APPROVED** [review]
Loading existing QA pairs and approved and version number to assets.

**Pull request merged**